lib/smci_report.rb

```diff
@@ -144,20 +144,16 @@ def run
                                                              middle_name: middle_name, institution: institution.strip)
             end
             # end check for institutions for this author
-            author_query = WebOfScience::QueryAuthor.new(author, symbolicTimeSpan: symbolicTimeSpan) # setup the WOS name query
-            uids = author_query.uids # now fetch all of the WOS_UIDs for the publications for this author by running the name search
-            logger.info(author_query.send(:author_query)) # log the name query being sent to WoS
+            name_query = WebOfScience::QueryName.new(author, symbolicTimeSpan: symbolicTimeSpan) # setup the WOS name query
+            uids = name_query.uids # now fetch all of the WOS_UIDs for the publications for this author by running the name search
+            logger.info(name_query.send(:name_query)) # log the name query being sent to WoS
```

Comment:
this class got renamed

Reply:
Can we avoid calling `send`?

Reply:
The `name_query` method is private, really never used externally, but I thought it was useful to log in this reporting class so we could see what was actually run (we feed the data to the class from a spreadsheet, so its a bit different than what happens in the code, where it comes from the database).